test/framework/alertmanager.go

```diff
@@ -441,44 +405,8 @@ func (f *Framework) WaitForAlertmanagerConfigToBeReloaded(ns, amName string, pre
 	return nil
 }
 
-type amAPICreateSilResp struct {
-	Status string             `json:"status"`
-	Data   amAPICreateSilData `json:"data"`
-}
-
-type amAPIPostAlertResp struct {
-	Status string `json:"status"`
-}
-
-type amAPICreateSilData struct {
-	SilenceID string `json:"silenceId"`
-}
-
-type amAPIGetSilResp struct {
-	Status string     `json:"status"`
-	Data   []amAPISil `json:"data"`
-}
-
-type amAPISil struct {
-	ID        string `json:"id"`
-	CreatedBy string `json:"createdBy"`
-}
-
-type amAPIStatusResp struct {
-	Status string          `json:"status"`
-	Data   amAPIStatusData `json:"data"`
-}
-
-type amAPIStatusData struct {
-	ClusterStatus *clusterStatus `json:"clusterStatus,omitempty"`
-	ConfigYAML    string         `json:"configYAML"`
-}
-
-type clusterPeer struct {
-	Name    string `json:"name"`
-	Address string `json:"address"`
-}
+type postSilencesOKBody struct {
```

Comment:
It's ok to import the struct from `github.com/prometheus/alertmanager/api/v2/client/silence` rather than duplicating it since we already depend on the `github.com/prometheus/alertmanager` module anyway.